samples/chat-with-custom-auth/csharp/FunctionApp/local.settings.sample.json

```diff
@@ -6,10 +6,11 @@
     "FUNCTIONS_WORKER_RUNTIME": "dotnet",
     "AzureSignalRConnectionString": "<signalr-connection-string>",
     "AzureSignalRServiceTransportType": "Transient",
-    "IssuerSigningKey": "<issuer-signing-key>"
+    "IssuerSigningKey": "bXlmdW5jdGlvbmF1dGh0ZXN0"
```

Comment:
hide this?

Reply:
This is a base64 encoded for "myfunctionauthtest". Users should need this string.
https://github.com/Azure/azure-functions-signalrservice-extension/blob/2eefea46113cc72b99e5aba60b1203890ba1de19/samples/chat-with-custom-auth/csharp/FunctionApp/Startup.cs#L33